docs/projects_we_track.md

```diff
@@ -568,3 +568,129 @@ The tool also checks for CVE issues and security advisories related to the CMS/f
+
+</p>
+<p align="right"><a href="https://github.com/be-secure/openc2-lycan-python"> LEARN MORE >> </a></p></p>
+
+
+
+<h2><b>powerfulseal</b></h2>
+<!-- <img src="./assets/images/projects-logos/xxxx" alt="no image"  height="100px"  width="100px" align="left"> <br> -->
+<b></b><p>
+
+A powerful chaos engineering tool for Kubernetes clusters.
+PowerfulSeal injects failure into your Kubernetes clusters, so that you can detect problems as early as possible. It allows for writing scenarios describing complete chaos experiments.
+
+</p>
+<p align="right"><a href="https://github.com/be-secure/powerfulseal"> LEARN MORE >> </a></p></p>
+
+
+
+<h2><b>mkdocs-material</b></h2>
+<!-- <img src="./assets/images/projects-logos/xxxx" alt="no image"  height="100px"  width="100px" align="left"> <br> -->
+<b></b><p>
+
+Material for MkDocs is a theme for MkDocs, a static site generator geared towards (technical) project documentation. If you're familiar with Python, you can install Material for MkDocs with pip, the Python package manager. If not, we recommended using docker.
+
+</p>
+<p align="right"><a href="https://github.com/be-secure/mkdocs-material"> LEARN MORE >> </a></p></p>
+
+
+
+
+<h2><b>ros_comm</b></h2>
+<!-- <img src="./assets/images/projects-logos/xxxx" alt="no image"  height="100px"  width="100px" align="left"> <br> -->
+<b></b><p>
+
+ROS communications-related packages, including core client libraries (roscpp, rospy, roslisp) and graph introspection tools (rostopic, rosnode, rosservice, rosparam).
+
+</p>
+<p align="right"><a href="https://github.com/be-secure/ros_comm"> LEARN MORE >> </a></p></p>
+
+
+
+
+
+<h2><b>emacs-secondmate</b></h2>
+<!-- <img src="./assets/images/projects-logos/xxxx" alt="no image"  height="100px"  width="100px" align="left"> <br> -->
+<b></b><p>
+
+An open-source, mini imitation of GitHub Copilot using EleutherAI GPT-Neo-2.7B (via Huggingface Model Hub) for Emacs.
+
+This is a much smaller model so will likely not be as effective as Copilot, but can still be interesting to play around with!
+
+</p>
+<p align="right"><a href="https://github.com/be-secure/emacs-secondmate"> LEARN MORE >> </a></p></p>
+
+
+
+
+<h2><b>asciinema</b></h2>
+<!-- <img src="./assets/images/projects-logos/xxxx" alt="no image"  height="100px"  width="100px" align="left"> <br> -->
+<b></b><p>
+
+asciinema project is built of several complementary pieces:
+
+command-line based terminal session recorder, asciinema,
+website with an API at asciinema.org,
+javascript player
+
+When you run asciinema rec in your terminal the recording starts, capturing all output that is being printed to your terminal while you’re issuing the shell commands. When the recording finishes (by hitting Ctrl-D or typing exit) then the captured output is uploaded to asciinema.org website and prepared for playback on the web.
+</p>
+<p align="right"><a href="https://github.com/be-secure/asciinema"> LEARN MORE >> </a></p></p>
+
+
+
+<h2><b>deploy</b></h2>
```

Comment:
@jobyko  I believe this talks about ansistrano/deploy & ansistrano/rollback, suggest you to change the headings accordingly.